base/cloudbuild.yaml.in

```diff
@@ -0,0 +1,8 @@
+steps:
+  - name: gcr.io/cloud-builders/docker
+    args: ['build', '--tag=${IMAGE}', './base']
+    id: BUILD
+  - name: gcr.io/gcp-runtimes/structure_test
+    args: ['--image', '${IMAGE}', '-v', '--config', '/workspace/test/test_config.json']
```

Comment:
since this is a cloud build step, this is run inside of an intermediate container that's constructed from all the previous steps. as per the cloud build spec, everything in the project is copied into `/workspace` inside this container, so we have to pass the full path since the structure test binary is also running inside of this same container.